package.json

```diff
@@ -24,12 +24,16 @@
     "url": "https://github.com/slack-samples/bolt-js-assistant-template/issues"
   },
   "dependencies": {
-    "@slack/bolt": "^4.4.0",
     "dotenv": "~17.2.1",
+    "@huggingface/inference": "^4.7.1",
```

Comment:
```suggestion
```

🪓 suggestion: We can remove reference to Hugging Face altogether if the code isn't active?